src/batch-client/index.ts

```diff
@@ -127,9 +127,16 @@ function normalizeMessage(
 	{ origin, jwtToken }: { jwtToken?: string; origin?: string }
 ) {
 	const normalizedMessage = normalize(MessageInfo)(message);
-	normalizedMessage.attributes =
-		normalizedMessage.attributes &&
-		mapValuesDeep(normalizedMessage.attributes, coerceStringToBoolean);
+	const { isSigned, isEncrypted } = normalizedMessage; // GetMsgResponse itself returns `isSigned` and `isEncrypted`. Let's use them as defaults.
```

Comment:
most probably we are getting this information because we have server side smime enabled
we will be disabling it, so this fix will not be valid in that case

Reply:
Awe.. didn't know that. Will have to figure out if there are any other solution for this.

Reply:
**UPDATE:**
I disabled smime zimlet which was enabled on an a/c where this issue was happening. And GetMsgResponse started returning correct data. So, this code change is not required at all.